src/public/modules/forms/admin/controllers/activate-form-modal.client.controller.js

```diff
@@ -98,8 +96,9 @@ function ActivateFormController(
       vm.esrvcIdStatus = 1
     })
 
-    if (authType === 'SP' && esrvcId !== '') {
-      return SpcpValidateEsrvcId(target, authType, esrvcId)
+    if ((authType === 'SP' && esrvcId !== '') || authType === 'MyInfo') {
```

Comment:
is the difference in handling for SP vs MyInfo forms required?

Reply:
good catch. i'll edit this out!